notebooks/en/phoenix_observability_on_hf_spaces.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>would be nice to show a screenshot or two about the features IMO </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Agreed! just added</p>